src/api/BioLink.js

```diff
@@ -254,51 +250,53 @@ export async function getNeighborhood(nodeId, nodeType) {
     rows: 100
   };
 
-  if (nodeType !== 'gene' && nodeType !== 'variant') {
-    const graphResponse = await axios.get(graphUrl, { params });
-    const graphResponseData = graphResponse.data;
+  const graphResponse = await axios.get(graphUrl, { params });
+  const graphResponseData = graphResponse.data;
 
-    // console.log('getNeighborhood', nodeId, graphUrl);
-    // console.log(JSON.stringify(graphResponseData, null, 2));
+  // console.log('getNeighborhood', nodeId, graphUrl);
+  // console.log(JSON.stringify(graphResponseData, null, 2));
 
-    if (graphResponseData.nodes) {
-      graphResponseData.nodes.forEach((node) => {
-        nodeLabelMap[node.id] = node.lbl;
-      });
-    }
-    if (graphResponseData.edges) {
-      graphResponseData.edges.forEach((edge) => {
-        if (edge.pred === 'subClassOf') {
-          if (edge.sub === nodeId) {
-            // console.log('Superclass Edge', edge.sub, edge.pred, edge.obj);
-            if (canUseSuperclassNode(nodeId, nodeType, edge.obj)) {
-              superclasses.push(edge.obj);
-            }
-          }
-          else if (edge.obj === nodeId) {
-            // console.log('Subclass Edge', edge.sub, edge.pred, edge.obj);
-            subclasses.push(edge.sub);
-          }
-          else {
-            // console.log('Unexpected edge', nodeId, edge.sub, edge.pred, edge.obj);
+  if (graphResponseData.nodes) {
+    graphResponseData.nodes.forEach((node) => {
+      nodeLabelMap[node.id] = node.lbl;
+    });
+  }
+  if (graphResponseData.edges) {
+    graphResponseData.edges.forEach((edge) => {
+      if (edge.pred === 'subClassOf') {
+        if (edge.sub === nodeId) {
+          // console.log('Superclass Edge', edge.sub, edge.pred, edge.obj);
+          if (canUseSuperclassNode(nodeId, nodeType, edge.obj)) {
+            superclasses.push(edge.obj);
           }
         }
-        else if (edge.pred === 'equivalentClass') {
-          // console.log('Equiv Edge', edge.sub, edge.pred, edge.obj);
+        else if (edge.obj === nodeId) {
+          // console.log('Subclass Edge', edge.sub, edge.pred, edge.obj);
+          subclasses.push(edge.sub);
+        }
+        else {
+          // console.log('Unexpected edge', nodeId, edge.sub, edge.pred, edge.obj);
```

Comment:
you have a few of these empty branch statements, but if its easier to leave them in for now, that is fine, as well